core/domain/exp_jobs_one_off.py

```diff
@@ -363,3 +363,60 @@ def map(item):
     @staticmethod
     def reduce(key, values):
         yield (key, values)
+
+
+class FallbacksToHintsOneOffJob(jobs.BaseMapReduceJobManager):
+    """Job that converts existing Fallbacks to Hints for explorations with
+    state schema version 10.
+
+    IMPORTANT: The GAE memcache must be flushed after running this job.
+    """
+    @classmethod
+    def entity_classes_to_map_over(cls):
+        return [exp_models.ExplorationModel]
+
+    @staticmethod
+    def map(item):
+        if item.deleted:
+            return
+        if item.states_schema_version != 10:
```

Comment:
Oh yeah you're right! I guess I thought it was included in the last release. Sorry, my bad. I'll remove this job.